pkg/controller/hostpathprovisioner/controller.go

```diff
@@ -237,6 +226,12 @@ func (r *ReconcileHostPathProvisioner) Reconcile(request reconcile.Request) (rec
 		return reconcile.Result{}, err
 	}
 
+	// Checks that only a single HPP instance exists
```

Comment:
This should be the first thing you do, before anything else.

Reply:
Done